mathics_django/doc/django_doc.py

```diff
@@ -11,13 +11,14 @@
 from mathics import settings
 from mathics.doc.common_doc import (
     DocChapter,
-    DocGuideSection,
+    DocPart,
+    DocSection,
+    DocSubsection,
     DocTest,
     DocTests,
     DocText,
     Documentation,
-    Tests,
-    XMLDoc,
+    DocumentationEntry,
     gather_tests,
```

Comment:
Something in my environment is telling me that `gather_tests` is no longer needed. 

Some thing for `import re` on line 8.